articles/quickstart/native/react-native-vnext/files/app.md

```diff
@@ -0,0 +1,54 @@
+---
+name: app.js
+language: javascript
+---
+
+```javascript
+import React from 'react';
+import {Button, Text, View} from 'react-native';
+import {useAuth0, Auth0Provider} from 'react-native-auth0';
+
+const Home = () => {
+  const {authorize, clearSession, user} = useAuth0();
+
+  const onLogin = async () => {
+    try {
+      await authorize();
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const onLogout = async () => {
+    try {
+      await clearSession();
+    } catch (e) {
+      console.log('Log out cancelled');
+    }
+  };
+
+  const loggedIn = user !== undefined && user !== null;
+
+  return (
+    <View style={styles.container}>
```

Comment:
You're right, I'll remove

Reply:
I guess we can rather add the styles because without them the UI doesn't look great
![Screenshot_1661469133](https://user-images.githubusercontent.com/15910425/186784519-f49d6c10-049c-46a6-8a6a-0512d4f271be.png)